### PR TITLE
fix: truncate time in plumed output to 1 fs

### PR DESCRIPTION
**Pull request by jmbuhr**
and regenerate examples from integration tests.

And copy plumed.dat input file to setup directory.

### CODE REVIEW

**Review by KRiedmiller — APPROVED** [review]

File: src/kimmdy/parsing.py [hunk 1]
```diff
@@ -504,13 +504,30 @@ def write_plumed(d: Plumed_dict, path: Path) -> None:
...
-            for k, v in zip(colnames, values):
+            values = l.strip().split()
+            time = values[0]
+            # time is in ps
+            # but needs to be truncated to
+            # 3 decimal places (1fs) to avoid
+            # floating point errors
+            # find the . :
+            i = time.index(".")
+            d["time"].append(float(time[: i + 4]))
```
Comment (KRiedmiller) [inline src/kimmdy/parsing.py:528]:
```suggestion
            d["time"].append(round(float(time), 3))
```

Comment (jmbuhr) [reply src/kimmdy/parsing.py:528]: true, that's a little less bespoke.

Comment (jmbuhr) [reply src/kimmdy/parsing.py:528]: Are you already making this change somewhere?

Comment (KRiedmiller) [reply src/kimmdy/parsing.py:528]: No, this tab was not refreshing itself, and looked like not yet merged.

**Pull request merged**